swagger.json

```diff
@@ -175,6 +175,22 @@
           "type" : "string",
           "description": "What groups should be able to access the resource in Access environments"
         },
+        "contains" : {
```

Comment:
Is there a reason we need to support both contains and containedBy?  Having two relationships that can be set greatly increases complexity in order to make consistency guarantees.